includes/class-scd-ext-access-control.php

```diff
@@ -49,9 +49,9 @@ class Scd_Ext_Access_Control {
 public function __construct(){
 
 	// set a formatted  message shown to user when the content has not yet dripped
-	$defaultMessage = __( 'This lesson will become available on [date].', 'sensei-content-drip' ) ;
-	$settingsMessage =  Sensei_Content_Drip()->settings->get_setting( 'scd_drip_message' ) ;
-	$this->message_format = empty( $settingsMessage ) ? $defaultMessage : $settingsMessage ;
+    $default_message = 'This lesson will become available on [date].';
+    $settings_field =  'scd_drip_message';
```

Comment:
rename this to `$settings_field_name`, or, better, use a string literal and remove the variable.